### DISCUSSION TITLE
[Feature request] Integrate with Google Drive

### DISCUSSION THREAD

**Discussion by chrisvaneijk** [PaulWoitaschek/Voice#1416] [Ideas]
As a user I want to be able to add Google Drive folder, so I can play my audiobook directly from Google Drive

**Comment by evra** [comment]
Having this feature can also enable easy Integration with chromecast.

**Comment by innesst** [comment]
"easy" 😂

**Comment by brianredbeard** [comment]
Duplicate of:
 - #229
 - #855 
 - #884

**Comment by bthoem** [comment]
This would be a wonderful feature - to be able to stream my audiobooks directly from Google Drive, with synchronization of reading position. Tx.

**Comment by Shadesss** [comment]
This would be amazing!
